lte/gateway/deploy/roles/trfserver/tasks/main.yml

```diff
@@ -98,6 +97,31 @@
     - iperf3_3.9-1_amd64.deb
   when: preburn
 
+- name: configure dhcpd
```

Comment:
Could this be move to preburn? If so, please add a `TODO` comment.

Reply:
doen